coffeecard/CoffeeCard.Models/Entities/Ticket.cs

```diff
@@ -13,7 +13,10 @@ public class Ticket
 
         public int ProductId { get; set; }
 
-        public bool IsUsed { get; set; }
```

Comment:
On second thought, I think we should play this safe and not delete anything until. Mark it as obsolete (true) for now. That way, nothing can reference it without breaking the build, but we preserve the data